slick/src/main/scala/akka/stream/alpakka/slick/javadsl/Slick.scala

```diff
@@ -164,4 +221,8 @@ object Slick {
   private def toDBIO[T](javaDml: JFunction[T, String]): T => DBIO[Int] = { t =>
     SQLActionBuilder(javaDml.asScala(t), SetParameter.SetUnit).asUpdate
   }
+
+  class JDBIO[T] {
+    def map[B](f: T => B): JDBIO[B] = ???
+  }
```

Comment:
What is this, and why does it throw, Isn't DBIO a type in slick?

Reply:
This was an attempt to create a Java version of Slick's `DBIO` so the additional `mapper` would not be necessary in the Java API, but gave up on this solution after trying it out for a while. 
Removed this code.